### PR TITLE
Document the CDL implementation

### PR DESCRIPTION
**Pull request by hackartisan**

### CODE REVIEW

File: docs/technical/cdl.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Controlled Digital Lending (CDL)
+
+With Controlled Digital Lending we allow digital access to a resource while ensuring that access does not exceed the access that would be permitted to the physical item. To achieve this, an item selected for CDL is sequestered physically, then only a single user can access it digitally at any given time. If two copies of an item are sequestered, then two users can access the digital copy, et cetera.
+
+In Figgy the CDL access business logic can be found in `app/services/cdl/` but its orchestration ranges through the uv_manager javascript, the viewer controller, and the cdl controller. Of course authorization logic can also be found in `ability.rb`. Bibdata provides access to data needed from Alma.
+
+What follows is a fairly detailed step-through of CDL access logic.
+
+- when figgy javascript goes to load the viewer, it submits a HEAD request to the manifest. If a 401 unauthorized is received, figgy redirects to /viewer/auth.
```
Comment (tpendragon) [inline docs/technical/cdl.md:9]: Is this still true after  our graphql refactor?

File: docs/technical/cdl.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - See the current sequence diagram: https://github.com/pulibrary/figgy/pull/5467/files?short_path=1162079#diff-11620797c34acf946659078dc8c9305a790f62e0e7da9e056e1a37871c377d96
+- the viewer controller's auth action uses the ChargeManager to look for CDL-eligigle items
```
Comment (tpendragon) [inline docs/technical/cdl.md:11]: Typo: eligible

**Review by tpendragon — CHANGES_REQUESTED** [review]

File: docs/technical/cdl.md (hunk 1 above)
Comment (hackartisan) [reply docs/technical/cdl.md:9]: nope!

**Comment by hackartisan** [comment]
@tpendragon ready for another look.

**Review by tpendragon — APPROVED** [review]

**Pull request merged**